### PR TITLE
added typography

### PR DESCRIPTION
**Pull request by shaksham08**
Added Typography component

image
image

### CODE REVIEW

File: components/typography/index.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        <a href="../../docs/index.html">Components</a>
+        <ul class="sidebar__list">
+          <li class="sidebar__item">
+            <a href="../alerts/index.html">Avatars</a>
+            <a href="../alerts/index.html">Alerts</a>
+            <a href="../alerts/index.html">Badges</a>
+            <a href="../button/index.html">Buttons</a>
+            <a href="../cards/index.html">Cards</a>
+            <a href="../images/index.html">Images</a>
+            <a class="active" href="#">Typography</a>
```
Comment (tusharkandpal) [inline components/typography/index.html]: `href` attribute can be `./index.html` instead of #

**Review by tusharkandpal — APPROVED** [review]
just one comment below. else is good to merge! 👍🏻

**Review by ajit-sharmaGH — APPROVED** [review]
Looks good to me, you can go ahead

**Review by sushil-mod — APPROVED** [review]
good to merge !

**Pull request merged**